lib/resources/TransferReversals.js

```diff
@@ -2,19 +2,8 @@
 
 const StripeResource = require('../StripeResource');
 
-/**
- * TransferReversals is a unique resource in that, upon instantiation,
- * requires a transferId, and therefore each of its methods only
- * require the reversalId argument.
- *
- * This streamlines the API specifically for the case of accessing reversals
- * on a returned transfer object.
- *
- * E.g. transferObject.reversals.retrieve(reversalId)
- * (As opposed to the also-supported stripe.transfers.retrieveReversal(transferId,
- * reversalId))
- */
 module.exports = StripeResource.extend({
-  path: 'transfers/{transferId}/reversals',
+  path: 'transfers/{id}/reversals',
```

Comment:
this should be `transferId` right?

Reply:
@rattrayalex-stripe Not as per OpenAPI, but we can override if necessary?

Reply:
ahhh interesting

Reply:
I think passing openapi's view of the world through is fine, carry on